### DISCUSSION TITLE
unable to find repo to add to the kitty FAQ

### DISCUSSION THREAD

**Discussion by jahway603** [kovidgoyal/kitty#5691] [Q&A]
Where is the repo for the FAQ located at https://sw.kovidgoyal.net/kitty/faq/? I would like to contribute to it, but I am unable to find it. Thanks.

**Comment by page-down** [comment]
> Where is the repo for the FAQ ...

The documentation source code is right here.
https://github.com/kovidgoyal/kitty/blob/master/docs/faq.rst
